_docs/_latest/configure/scanner/rules.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: Defining Rules
-order: 103
+order: 305
 ---
 
 # {{ page.title }}
 
 This page describes how to define rules for Forseti Scanner.
 
+---
+
```

Comment:
This file and all of it's rule definitions below need reformatted to the common format (name, description, type, values, notes, examples).

Reply:
Updated.